config/roguelike_dungeons/settings/loot_armor.json

```diff
@@ -0,0 +1,126 @@
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 8},
+		{"data": {"name": "battlegear2:shield.hide", "meta": 0}, "weight": 15}
+    ]
+    },
+    {"level": 2, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubychestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubyhelmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubyleggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.rubyboots", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphireboots", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphirechestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphirehelmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "ProjRed|Exploration:projectred.exploration.sapphireleggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_boots", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_chestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_helmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_leggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 8},
+		{"data": {"name": "battlegear2:shield.gold", "meta": 0}, "weight": 15}
+      ]
+    },
+    {"level": 3, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "Railcraft:armor.steel.boots", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.helmet", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.legs", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.plate", "meta": 0}, "weight": 15},
+		{"data": {"name": "minecraft:diamond_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "battlegear2:shield.iron", "meta": 0}, "weight": 15}
```

Comment:
battlegear2 is going to be removed, give out a fully built Tinker's Defense shield here instead.